frontend/helpers/next-helpers.ts

```diff
@@ -18,17 +20,23 @@ export function serverSidePropsWrapper<T extends { [key: string]: any }>(
          ...context.params,
          ...context.query,
       });
-      console.log({
-         'x-forwarded-host': context.req.headers['x-forwarded-host'],
-         'x-ifixit-forwarded-host':
-            context.req.headers['x-ifixit-forwarded-host'],
-      });
       return logAsync('getServerSideProps', () =>
          getServerSidePropsInternal(context)
-      ).catch((err) => {
-         setSentryPageContext(context);
-         throw err;
-      });
+      )
+         .then((result) => {
+            if (CACHE_DISABLED || context.query._vercel_no_cache === '1') {
+               context.res.setHeader(
+                  'Cache-Control',
+                  'no-store, no-cache, must-revalidate, stale-if-error=0'
+               );
+               clearCache();
```

Comment:
We'll actually want to do this line *before* we call the `getServerSidePropsInternal` otherwise, we've already used the cache by this point. That said... seems a bummer that we're not caching strapi in dev at all. I don't remember the motivation for that... do you?

Reply:
This is where it was added I think: https://github.com/iFixit/react-commerce/pull/659 @danielcliu-ifixit

Reply:
I ended up clearing the cache before and after. Let me know if you think that is incorrect, I wasn't sure.